### PR TITLE
Add summarize-v2-outputs dev tool

### PR DESCRIPTION
**Pull request by F88**
## Summary

- Adds `summarize-v2-outputs`, a dev tool that catalogs v2 pipeline outputs per source — a "manifest of data sources". Reads `public/data-v2/<prefix>/{data,insights,shapes}.json` plus `global/insights.json` and reports file sizes, per-bundle entity counts, and per-section detail (feed-info, agencies, routes, stops, trip-patterns, i18n-coverage, periods, shapes-volume, trip-volume, global-insights).
- Layered as entry script -> main lib -> per-bundle sub libs; each sub lib owns one bundle type's analyze + render.
- Also includes a small unrelated config fix (`ae32806b`): reorder `routeTypes` for consistency in `data-source-settings.ts` source groups.

Related: #212 (this tool is the reference groundwork for the proposed `DataSourceCatalogBundle`).

## Test plan

- [x] `npx vitest run pipeline/scripts/dev/dev-lib/__tests__/` — 158 tests pass
- [x] `npx tsc --noEmit` — no errors
- [x] `npx eslint` / `npx prettier --check` on changed files — clean
- [x] `npx tsx pipeline/scripts/dev/summarize-v2-outputs.ts` — runs and produces the report
- [x] `npx tsx pipeline/scripts/dev/summarize-v2-outputs.ts --list-sections` — lists all sections

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: src/config/data-source-settings.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ const settings: SourceGroup[] = [
   {
     id: 'toei-train',
     prefixes: ['toaran'],
-    routeTypes: [0, 1, 2],
+    routeTypes: [0, 2, 1],
```
[the remark this answers is not in the document]
Comment (F88) [reply src/config/data-source-settings.ts:40]: This is an intentional change.

**Comment by F88** [comment]
## Review response

Thanks for the review. All comments were verified against the actual code; here is how each was handled.

### Fixed -- commit ad9ca88d
- **`--list-sections` ENOENT**: now returns before `listSourcePrefixes()`, so it works without `public/data-v2`.
- **Stale `tripsMedian`** (README + `dev-tools.ts`): replaced with `tripsTotal` / `tripsMax`.
- **Stale `global-insights` README description**: now reflects wp/cn coverage.
- **Orphaned TSDoc** in `v2-data-summary.ts`: moved to `buildI18nCoverageSummary`.

### Fixed -- commit 53c533b6
- **`FileSizeStats.insights` -> `number | null`** (+ `measureSource` measures a missing `insights.json` as `null`, totals use the nullable sum): a missing file now renders `-`, distinct from a real `0 B` file -- matching `shapes` and the documented contract.
- **`measureSource`**: shapes size reuses the in-memory buffer length; the redundant `statSync` / `statSizeIfExists` helper is removed.
- **`serviceGroupCount`** now derives from the required `serviceGroups` section, not the optional `tripPatternStats` group count.
- **`sourcesWithInsights` / `sourcesMissingInsights`**: `V2InsightsSummary` gained `bundlePresent`; both now track actual bundle presence rather than `tripsTotal !== null`.
- **`buildGlobalInsightsBundleCounts(null)`** returns `{}` (was `{ stopGeo: 0 }`) so an absent bundle is distinguishable from a present-but-empty one; the `GlobalInsightsBundleSummary` TSDoc is corrected to match.

Two `v2-insights-summary` tests turned out to encode the old buggy behavior / use malformed fixtures (serviceGroups != tripPatternStats); those were corrected, and edge-case coverage was added for the missing-bundle / nullable-size paths.

### Declined -- `countSection` / `sumNullable` duplication
The per-bundle sub libs are **intentionally self-contained** -- each owns exactly one bundle type's analyze + render, with no cross-sub-lib imports (stated in the module TSDoc, e.g. `v2-insights-summary.ts`: "Kept locally to keep this sub lib self-contained"). The ~10-line helpers are deliberately duplicated to preserve that boundary. The Gemini comment's own cited reference -- "Avoid creating shared helper functions for small code blocks (~10 lines)" -- argues the same way.

### Left as-is -- `data-source-settings.ts` twr-rinkai ordering
Out of scope for this PR; the `twr-rinkai` `routeTypes` ordering is unchanged here.

All checks green: `tsc` / ESLint / Prettier / 160 tests.
